### PR TITLE
added badges

### PR DESCRIPTION
**Pull request by 3scava1i3r**
##### Description

 added badges for better visualization and better social media reach
and deleted the non responsive image at the end of the README.md file
##### Refers/Fixes

#7225 
##### Testing
Readme file edit would not need testing right?

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,14 +4,43 @@
...
+   <img alt="GitHub contributors" src="https://img.shields.io/github/contributors/gitcoinco/web">
+  </a>
+  <a href="">
+    <img alt="GitHub" src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg">
+  </a>
+  <a href="">
+  <img alt="GitHub top language" src="https://img.shields.io/github/languages/top/gitcoinco/web">
+  <a/>
+  <a href="https://circleci.com/gh/facebook/flow/tree/master">
+    <img src="https://circleci.com/gh/facebook/flow/tree/master.svg?style=shield" alt="CircleCI" />
```
Comment (3scava1i3r) [inline docs/README.md]: deleted this now

File: docs/README.md [hunk 2]
```diff
@@ -211,6 +237,3 @@ Note:
 
 - [Code - GNU AFFERO GENERAL PUBLIC LICENSE](../LICENSE)
 - [Creative assets -- Attribution-NonCommercial-NoDerivatives 4.0 International](../app/assets/LICENSE.txt)
-
-<!-- Google Analytics -->
-<img src='https://ga-beacon.appspot.com/UA-102304388-1/gitcoinco/web' style='width:1px; height:1px;' >
```
Comment (thelostone-mc) [inline docs/README.md:216]: why was this removed ?

Comment (3scava1i3r) [reply docs/README.md:216]: the image is no longer available and shows a dead link so please find and alternative but till then i removed it
